libs/entitlement/qa/build.gradle

```diff
@@ -7,23 +7,12 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-apply plugin: 'elasticsearch.base-internal-es-plugin'
 apply plugin: 'elasticsearch.internal-java-rest-test'
 // Necessary to use tests in Serverless
 apply plugin: 'elasticsearch.internal-test-artifact'
 
-esplugin {
-  name 'entitlement-qa'
-  description 'A test module that triggers entitlement checks'
-  classname 'org.elasticsearch.test.entitlements.EntitlementsCheckPlugin'
-}
-
 dependencies {
-  clusterPlugins project(':qa:entitlements')
+  clusterPlugins project(':libs:entitlement:qa:entitlement-negative-check')
```

Comment:
I know we've been referring to these as negative and positive checks, but perhaps it would be more clear to use "allowed" and "denied"? So simply `entitlement-allowed` and `entitlement-denied`?

Reply:
Renamed